pandas/core/dtypes/common.py

```diff
@@ -16,10 +16,10 @@
     ABCSparseArray, ABCSparseSeries, ABCCategoricalIndex, ABCIndexClass,
     ABCDateOffset)
 from pandas.core.dtypes.inference import (  # noqa:F401
-    is_bool, is_integer, is_hashable, is_iterator, is_float,
-    is_dict_like, is_scalar, is_string_like, is_list_like, is_number,
-    is_file_like, is_re, is_re_compilable, is_sequence, is_nested_list_like,
-    is_named_tuple, is_array_like, is_decimal, is_complex, is_interval)
+    is_bool, is_integer, is_float, is_number, is_decimal, is_complex,
+    is_re, is_re_compilable, is_dict_like, is_string_like, is_file_like,
+    is_list_like, is_nested_list_like, is_sequence, is_named_tuple,
+    is_hashable, is_iterator, is_array_like, is_scalar, is_interval)
```

Comment:
This is somewhat of an artefact of the version with `is_ordered_list_like`, where I tried to group these methods by similarity (i.e. scalar dtypes, regexes, containers), but I decided to keep it because I think it helps. Can revert that part of course

Reply:
yes, on any change, pls try to do the minimal changeset. This will lessen reviewer burden and make things go faster.

Reply:
Fine as is for now.

Reply:
ok for now, but generally pls don't change unrelated things.